src/main/java/uk/gov/justice/digital/config/JobParameters.java

```diff
@@ -47,6 +47,10 @@ public Duration getKinesisReaderBatchDuration() {
         return Durations.seconds(parsedDuration);
     }
 
+    public String getRawPath() {
+        return getMandatoryProperty("s3.raw.path");
```

Comment:
If this property is specific to the raw job perhaps `raw.s3.path` is a better way to go here.